lib/agent0/agent0/hyperdrive/policies/smart_long.py

```diff
@@ -131,7 +131,7 @@ def action(
         # only open a long if the fixed rate is higher than variable rate
         if (interface.fixed_rate - interface.variable_rate) > self.risk_threshold and not has_opened_long:
             # calculate the total number of bonds we want to see in the pool
-            total_bonds_to_match_variable_apr = interface.bonds_given_shares_and_rate(
+            total_bonds_to_match_variable_apr = interface.calculate_bonds_given_shares_and_rate(
```

Comment:
Didn't you rename these to `calc_...` or was that a different PR?

Reply:
that's in a follow-up PR reformatting the API.  And tbh I'm not sure if I'll even stick with it. I want it to be consistent across languages, but I dislike how it is in SOL & RS.